src/index.js

```diff
@@ -1,18 +1,38 @@
 import express from 'express';
+import passport from 'passport';
+import session from 'express-session';
+import cors from 'cors';
+import logger from 'morgan';
 import apiRouter from './api/routes/index';
 import docsRouter from './api/routes/docs';
 import homeRouter from './api/routes/home';
-import register from './middlewares/register.app';
 import env from './configs/environments';
+// import register from './middlewares/register.app';
+
+import passportConfig from './middlewares/passport';
+import './helpers/cloudnary';
 
 const app = express();
 
-// Register middleware
-register(app);
+// register(app);
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(logger('dev'));
+
+app.use(express.static(`${__dirname}/public`));
+app.use(passport.initialize());
+passportConfig(passport);
+
+app.use(session({
+  secret: env.secret,
+  cookie: { maxAge: 60000 },
+  resave: false,
```

Comment:
@murediane why are you setting these cookies?

Reply:
They are removed now.